src/icon-manifest/supportedExtensions.ts

```diff
@@ -78,8 +79,8 @@ export const extensions: IFileCollection = {
     { icon: 'cobol', extensions: ['cbl', 'cob', 'cpy'], format: FileFormat.svg },
     { icon: 'codeclimate', extensions: ['codeclimate.yml'], light: true, filename: true, format: FileFormat.svg },
     { icon: 'coffeescript', extensions: [], languages: [languages.coffeescript], format: FileFormat.svg },
-    { icon: 'config', extensions: ['env', 'config'], languages: [languages.properties], format: FileFormat.png },
```

Comment:
As filename it's declared in `properties` language id.
https://github.com/Microsoft/vscode/blob/1.8.1/extensions/ini/package.json#L16